src/client/Microsoft.Identity.Client/ApiConfig/AbstractAcquireTokenParameterBuilder.cs

```diff
@@ -384,7 +384,7 @@ internal void ValidateAndCalculateApiId()
             CommonParameters.CorrelationId = CommonParameters.UseCorrelationIdFromUser ? CommonParameters.UserProvidedCorrelationId : Guid.NewGuid();
         }
 
-        internal void ValidateUseOfExpirementalFeature([System.Runtime.CompilerServices.CallerMemberName] string memberName = "")
+        internal void ValidateUseOfExperimentalFeature([System.Runtime.CompilerServices.CallerMemberName] string memberName = "")
```

Comment:
?

Reply:
This came from jenny's PR.
@jennyf19 do we wont to make this experimental?

Reply:
I don't think so. In any case, the experimental check is not made anywhere.

Reply:
It's a spelling change

Reply:
Ah yes, subtle :)